exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,14 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+
+function danti(a) {
+    return "HELOW";
+
+}
+console.log(danti() + " " + mentor1.toUpperCase() );
+console.log(danti() + " " + mentor2.toUpperCase() );
+console.log(danti() + " " + mentor3.toUpperCase() );
+console.log(danti() + " " + mentor4.toUpperCase() );
+console.log(danti() + " " + mentor5.toUpperCase() );
```

Comment:
The solution does not work, and you receive an error message!
Here are some threads to fix the error:
1. For all of our mentors' names to be in uppercase, you need to declare that in your function, for example: return "Hello" + n.toUpperCase();
2. You can call your function from the console.log and pass the mentors' variables as parameters.